OpenBCI_32bit_Library.cpp

```diff
@@ -130,19 +130,12 @@ char OpenBCI_32bit_Library::getCharSerial1(void) {
   return Serial1.read();
 }
 
-/**
-* @description While processing incoming multi byte messages these will turn
-*  true.
-* @return {boolean} - True if processing a message and false otherwise
-*/
-boolean OpenBCI_32bit_Library::isProcessingMultibyteMsg(void) {
-  return isProcessingIncomingSettingsChannel || isProcessingIncomingSettingsLeadOff || settingBoardMode || settingSampleRate;
-}
 
 /**
  * Used to abort a multipack message
  */
-void OpenBCI_32bit_Library::tryMultiAbort(void) {
+/*  Replaced by startMultiCharCmdTimer
```

Comment:
please remove old function, thanks to git history, we can just ditch this old code!